### DISCUSSION TITLE
Improvements Alert Component

### DISCUSSION THREAD

**Discussion by ryuhangyeong** [mantinedev/mantine#2082] [Feature requests]
## Etc

Thanks for making such a great library!

## Suggest

Since the close button belongs to the title, it can be displayed only if the title exists. But I think the close button should be outside the title. What do you think?

```tsx
{title && (
  <div className={classes.title}>
    <span id={titleId} className={classes.label}>
      {title}
    </span>

    {withCloseButton && (
      <CloseButton
        className={classes.closeButton}
        onClick={onClose}
        variant="transparent"
        size={16}
        iconSize={16}
        aria-label={closeButtonLabel}
      />
    )}
  </div>
)}
```

![image](https://user-images.githubusercontent.com/27342882/184267816-d89aad0b-4b9e-4daa-80a5-1e7e9e4d2442.png)

If you set an empty value to the title, you can create what you want, but I think the part marked in red is awkward.

**Comment by zhulien-ivanov** [comment]
This was covered here: #1348.

**Reply by ryuhangyeong** [reply]
Thanks~

**Comment by rtivital** [comment]
Fixed in 5.1.4

**Reply by ryuhangyeong** [reply]
Thanks~
